docs/plugins/harbor.md

```diff
@@ -0,0 +1,93 @@
+# harbor Plugin
+
+This plugin installs [harbor](https://goharbor.io/) in an existing Kubernetes cluster using the Helm chart.
+
+## Usage
+
+##### External storage
+
+- Postgresql: Set the `database.type` to `external` and fill the information in `database.external` section.
+- Redis: Set the `redis.type` to `external` and fill the information in `redis.external` section.
+
+##### Disk storage
+
+Please be sure to change the `storageClass` in the options of the config to an existing StorageClass. You can refer to this [document](https://github.com/goharbor/harbor-helm#configure-how-to-persist-data)
+
+#### Network config
+
+This plugin support `Ingress`, `ClusterIP`, `NodePort`, `LoadBalancer` ，You can give choice to your needs.
+
+##### TLS config
+- Use self-signed certificates
```

Comment:
Although here it isn't necessary, generally it looks cleaner if we have an empty line between a subtitle and the body text.